streaming/stream_session.hh

```diff
@@ -215,7 +215,7 @@ public:
      * @param ranges Ranges to retrieve data
      * @param columnFamilies ColumnFamily names. Can be empty if requesting all CF under the keyspace.
      */
-    void add_stream_request(sstring keyspace, dht::token_range_vector ranges, std::vector<sstring> column_families) {
+    void add_stream_request(sstring keyspace, dht::token_range_vector&& ranges, std::vector<sstring>&& column_families) {
```

Comment:
I don't think this affects exception safety in any way.

Reply:
If `_requests.emplace_back` fails to allocate, the `ranges` arg, that was already moved by the caller, will be destroyed, while with passing the rvalue reference the move will not take place in this case.
If allocation succeeded, then `std::move(ranges)` will take place in the ctor (hence emplace_back is preferred over push_back) which is noexcept at this point.